app/models/health/qualifying_activity.rb

```diff
@@ -324,6 +324,14 @@ def outreach?
       activity == 'outreach'
     end
 
+    def screening?
+      activity == 'sdoh_positive' || activity == 'sdoh_negative'
```

Comment:
to avoid some duplication
```suggestion
     positive_screen? || activity == 'sdoh_negative'
```